### ISSUE TITLE
Prodigy logs

### ISSUE THREAD

**Issue by eroux** [OpenPecha/prodigy-tools#4]
I tried to look at the logs of prodigy but couldn't really find anything useful in `journalctl -u prodigy_bdrc_crop_images.service`, I'm not sure where prodigy logs errors, info, etc. but it should be instructed to do so in a file in  `/usr/local/prodigy/logs/`

**Comment by ta4tsering** [comment]
Both error and info log of the bdrc_crop_images.py recipe is at the `/usr/local/prodigy/logs/bdrc_crop_images.log`.    
   
[logs path in recipe](https://github.com/OpenPecha/prodigy-tools/blob/main/recipes/bdrc_crop_images.py#L17)


<img width="544" alt="Screenshot 2023-01-12 at 10 26 32 AM" src="https://user-images.githubusercontent.com/43548581/211980442-ff1b530c-8444-4906-9b5d-1a24f6895b9e.png">


below is the log file

[bdrc_crop_images.log](https://github.com/OpenPecha/prodigy-tools/files/10398304/bdrc_crop_images.log)

**Comment by eroux** [comment]
yes, I'm aware of that, I was referring to the prodigy logs in general

**Comment by ta4tsering** [comment]
so I have introduced the `Environment=PRODIGY_LOOGING=verbose` in the `/etc/systemd/system/prodigy_bdrc_crop_images.service` as per the [Documentaion](https://prodi.gy/docs/install#debugging-logging) and when I do ` sudo journalctl -u prodigy_bdrc_crop_images.service` and go to the page, all the logs are available but I couldn't instruct it to write the logs at the `/usr/local/prodigy/logs`. I tried by adding `"log_file": "/usr/local/prodigy/logs/prodigy.log"` in the `/usr/local/prodigy/prodigy-tools/configuration/bdrc_crop_images.json` but after restarting the server. there is still no `prodigy.log` file in the `/usr/local/prodigy/logs`
below is the image of the page after the command `sudo journalctl -u prodigy_bdrc_crop_images.service`
<img width="739" alt="Screenshot 2023-01-16 at 4 17 35 PM" src="https://user-images.githubusercontent.com/43548581/212660545-20a38c05-ba11-45bc-8c42-aca9aa0cfbc9.png">


p.s I have tested all through making changes in the .service file at the server. if the addition of the `Environment=PRODIGY_LOGGING=verbose` looks good, then I will commit it in the `prodigy-tools/install-scripts/prodigy_bdrc_crop_images.service` and push the commit.

**Comment by eroux** [comment]
thanks for the investigation! I'm sure we can find a way to get the logs somehow. If you're rally stuck you can look on the Prodigy forum and ask a new question if you don't find anything, they seem to answer usually

**Comment by ta4tsering** [comment]
sure, I will post the question on the forum.
